src/App.tsx

```diff
@@ -2,51 +2,94 @@ import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
-
-export const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
-      data-cy="Nav"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
-        </div>
-      </div>
-    </nav>
-
-    <div className="section">
-      <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="tabs is-boxed">
-          <ul>
-            <li data-cy="Tab" className="is-active">
-              <a href="#/">Tab 1</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 2</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 3</a>
-            </li>
-          </ul>
+import classNames from 'classnames';
+import {
+  Navigate,
+  NavLink,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import { TabsPage } from './TabsPage';
+
+type Props = {
+  to: string,
+  text: string,
+};
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => (
+  <NavLink
+    className={({ isActive }) => classNames(
+      'navbar-item',
+      { 'is-active': isActive },
+    )}
+    to={`${to}`}
+  >
+    {text}
+  </NavLink>
+);
+
+export const App = () => {
+  return (
+    <>
+      {/* Also requires <html class="has-navbar-fixed-top"> */}
+      <nav
+        className="navbar is-light is-fixed-top is-mobile has-shadow"
+        data-cy="Nav"
+      >
+        <div className="container">
+          <div className="navbar-brand">
+            <PageNavLink to="/" text="Home" />
+            <PageNavLink to="/tabs" text="Tabs" />
+          </div>
         </div>
+      </nav>
+
+      <div className="section">
+        <div className="container">
+          <Routes>
+            <Route
+              path="/"
+              element={<h1 className="title">Home Page</h1>}
```

Comment:
```suggestion
              element={<h1 className="title">Home page</h1>}
```
test fails because of this